include/zmq.h

```diff
@@ -377,6 +384,23 @@ ZMQ_EXPORT const char *zmq_msg_gets (const zmq_msg_t *msg,
 #define ZMQ_GSSAPI_SERVICE_PRINCIPAL_NAMETYPE 91
 #define ZMQ_BINDTODEVICE 92
 
+#define ZMQ_PEER_EVENTS 93
```

Comment:
this should be in the DRAFT section as well - please move it and the block below wit the struct after line 619, where "DRAFT Socket options." is - and you can remove the ifdefs